### PR TITLE
giftcard calss

### PR DESCRIPTION
**Pull request by PurviShah06**
create giftcard class

### CODE REVIEW

**Comment by PriyaHirapara** [comment]
hi 
i got you todays work i will revied by tomarrow becoz i m busy in my class

File: src/test/java/GiftCardPage.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+public class GiftCardPage {
+
```
Comment (Divyakorat) [inline src/test/java/GiftCardPage.java:2]: looks good

Comment (PurviShah06) [reply src/test/java/GiftCardPage.java:2]: thank you

Comment (anjalipatel2422) [inline src/test/java/GiftCardPage.java:2]: plz enter detail

**Pull request merged**